src/urg_node_driver.cpp

```diff
@@ -643,6 +632,19 @@ void UrgNode::scanThread()
   }
 }
 
+bool UrgNode::sensorStatusOk(const std::string& status) const
+{
+  static std::string NOT_CONNECTED = std::string("not connected.");
+  static std::string RECEIVE_ERROR_MESSAGE = std::string("not connected.");
+  static std::string SENSOR_WORKS_WELL = std::string("Sensor works well.");
+  static std::string SENSOR_STABLE_NO_ERROR = std::string("Stable 000 no error.");
+  static std::string SENSOR_WORKING_NORMALLY = std::string("sensor is working normally");
+
+  return (status != NOT_CONNECTED && status != RECEIVE_ERROR_MESSAGE) &&
+         (status == std::string("Sensor works well.") || status == std::string("Stable 000 no error.") ||
+          status == std::string("sensor is working normally"));
```

Comment:
Was the intent to use your static variables?
```suggestion
         (status == SENSOR_WORKS_WELL) || status == SENSOR_STABLE_NO_ERROR) ||
          status == SENSOR_WORKING_NORMALLY));
```